src/pysdmx/io/xml/sdmx_two_one/writer/metadata_writer.py

```diff
@@ -0,0 +1,276 @@
+"""Module for writing metadata to XML files."""
```

Comment:
Similar comments about 1) long package names and 1) the fact that metadata might mean reference metadata to some people.

So, instead of:
`pysdmx.io.xml.sdmx_two_one.writer.metadata_writer`
we could have:
`pysdmx.io.xml.sdmx21.writer.structure`

Reply:
Changing it to structure, giving the new file structure makes more sense.